app/renderer/js/pages/preference/find-accounts.js

```diff
@@ -0,0 +1,71 @@
+'use-strict';
+
+const { remote, ipcRenderer } = require('electron');
+
+const { dialog } = remote;
+
+const BaseComponent = require(__dirname + '/../../components/base.js');
+
+class FindAccounts extends BaseComponent {
+	constructor(props) {
+		super();
+		this.props = props;
+	}
+
+	template() {
+		return `
+			<div class="settings-card certificate-card">
+				<div class="certificate-input">
+					<div>Organization URL</div>
+					<input class="setting-input-value" value=".zulipchat.com"/>
```

Comment:
Don't we just want the server input to have `zulipchat.com` instead of `.zulipchat.com`? cc @akashnimare @rishig.

Reply:
yeah